PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1903,6 +1904,7 @@ If you select advanced mode but do not edit files in the aforementioned language
 			<max-len-on-search-tip value="Your input may exceed the limit allowed and could have been truncated, and it won't be saved for the next session."/>
 			<max-len-on-save-tip value="The length of your input is very long and may not be saved for your next session."/>
 			<goto-setting-tip value="Find your setting here"/>
+			<tabbar-tabcompactlabellen-tip value="This is to limit the visible length of the document tab name and thus the tab size. The allowed range is 0 - 257 characters, with 0 meaning that the compacting is disabled. Shortened names are recognizable by trailing ellipses."/>
```

Comment:
"Shortened names are recognizable by trailing ellipses." is not necessary IMO - Users will understand after applying this settings.

I would suggest the following: "Truncates long tab names. Enter to apply the given value. Value range: 1-257 characters (0 disables truncation)."

Reply:
@donho 

> Please consider the following behaviour:
> 
> > While user remove the value completely, it should be considered as disabling the feature, and 0 should be restored.

Yes, I'm taking this into account for the next commit.